2022q4/wifi.adoc

```diff
@@ -0,0 +1,34 @@
+=== Wireless updates
+
+Links: +
+link:https://people.freebsd.org/~bz/wireless/[Bjoern's Wireless Work In Progress landing page] URL: link:https://people.freebsd.org/\~bz/wireless/[https://people.freebsd.org/~bz/wireless/] +
+
+Contact: Bjoern A. Zeeb <bz@FreeBSD.org>
+
+During the 4th quarter of 2022 not much work was publicly visible and admittingly slightly slow.
+Behind the scenes wireless work was happening on two fronts:
+
+ * work on 11n and 11ac and wpa bits,
+
+ * and continous work on more drivers and firmware, problems, testing, and filling more gaps for these.
+
+While the main development for newer standards and the Intel iwlwifi driver is sponsored by the FreeBSD Foundation, I find myself spending a lot of private time with Realtek rtw88 and rtw89, Atheros, and Mediatek mt76 (7921 and 7915) drivers now as well.
+Testing changes on bare metal and using bhyve VMs using passthru has become more time-consuming, with the amount of supported chipsets increasing, in order not to break other drivers.
+Even different (generations of) chipsets supported by the same driver, at times, behave different to the same change.
+
+A separate discussion was brought to me about the size of firmware added to the tree for the already existing drivers and the firmware for more drivers coming.
+The chicken-egg problem to solve is to have the firmware available on the release media as otherwise a lot of modern laptops will not have any sort of outside communications available at the time of install in their default ocnfiguration should we not commit it to main anymore.
+This will be a larger discussion to have to also solve firmware for other drivers, but that discussion will be for another day and place.
+
+Slightly belated I have started to flush LinuxKPI and 802.11 changes into the tree and of the year and that work will drag into early 2023 at which point more of the aforementioned remaining drivers will also hit the tree.
```

Comment:
s/belated/belatedly/

s/flush/push/ maybe?

s/and of the year/at the end of the year/